lib/config/presets/internal/default.ts

```diff
@@ -83,7 +83,8 @@ export const presets: Record<string, Preset> = {
     ],
   },
   pinDependencies: {
-    description: 'Pin dependency versions for <code>dependencies</code>',
+    description:
+      'Pin dependency versions where depType=dependencies. Usually applies only to non-dev dependencies in <code>package.json</code>',
```

Comment:
```suggestion
      'Pin dependency versions where <code>depType=dependencies</code>. Usually applies only to non-dev dependencies in <code>package.json</code>',
```

We can add the `<code>` formatting to `depType=dependencies` as well.